src/diagonal.works/b6/ingest/features.go

```diff
@@ -303,6 +312,17 @@ func (p *PathFeature) AllPoints(byID b6.LocationsByID) ([]s2.Point, error) {
 	return points, nil
 }
 
+func (p *PathFeature) References() []b6.Reference {
+	references := make([]b6.Reference, 0, p.Len())
+	for i := 0; i < p.Len(); i++ {
+		if id, ok := p.PointID(i); ok {
+			references = append(references, &indexedReference{id.FeatureID(), i})
```

Comment:
Did we say this was one of the cases where you were going to switch to struct, not a pointer - or do you rely on it being a pointer later? Otherwise it's adding an extra 8 bytes/reference for the pointer.

Reply:
i have to make it a reference to overwrite the index

Reply:
see setindex in references.addfeature()

Reply:
(this is my accessibility b/ !!)

i think i still dont understand y we do that tho.. but i ll follow up w u offline